packages/graphql-transformer-core/src/collectDirectives.ts

```diff
@@ -47,7 +47,7 @@ export function collectDirectives(sdl: string): DirectiveNode[] {
   return directives;
 }
 
-export function collectDirectivesByTypeNames(sdl: string): Object {
+export function collectDirectivesByTypeNames(sdl: string) {
```

Comment:
Should this be part of this change?

Reply:
Yeah we use this method in the api category and removing this allows TS to infer the type from the return value